selenium/src/test/java/com/deque/html/axecore/selenium/AllowedOriginsTest.java

```diff
@@ -39,7 +39,7 @@ private Object getAllowedOrigins() {
 
   @Before
   public void setup() throws Exception {
-    webDriver = new ChromeDriver(new ChromeOptions().setHeadless(true));
+    webDriver = new ChromeDriver(new ChromeOptions().addArguments("--headless"));
```

Comment:
Result of updating Selenium to 4.13.0. They removed `setHeadless()`. 

Release Notes: https://github.com/SeleniumHQ/selenium/blob/trunk/java/CHANGELOG#L35
Commit: https://github.com/SeleniumHQ/selenium/commit/dbcad078e520c80605b4fce856ceda30be33fc93